contrib/test_jenkins.sh

```diff
@@ -953,14 +973,17 @@ run_client_server() {
 
 run_ucp_client_server() {
 	echo "==== Running UCP client-server  ===="
-	run_client_server
+	run_client_server $(get_rdma_device_ip_addr)
+	run_client_server $(get_non_rdma_ip_addr)
 
 	rm -f ./ucp_client_server
 }
 
 run_io_demo() {
-	server_ip=$(get_rdma_device_ip_addr)
-	if [ "$server_ip" == "" ]
+	server_rdma_addr=$(get_rdma_device_ip_addr)
+	server_nonrdma_addr=$(get_non_rdma_ip_addr)
+
+	if [ "$server_rdma_addr" == "" ] && [ "$server_nonrdma_addr" == ""  ]
```

Comment:
`if [ -n "$server_rdma_addr" ] && [ -n "$server_nonrdma_addr" ]`

Reply:
if both are non-zero (-n) then we shouldn't return, we should run the test.
did you mean -z ?

Reply:
yes, I've meant -z